docs/features/routing.rst

```diff
@@ -69,11 +69,26 @@ In order to change this you can specify on a per Route basis the following setti
 
   "RouteIsCaseSensitive": true
 
-This means that when Ocelot tries to match the incoming upstream URL with an upstream template the evaluation will be case sensitive. 
+This means that when Ocelot tries to match the incoming upstream URL with an upstream template the evaluation will be case sensitive.
+
+.. _routing-embedded-placeholders:
+
+Embedded Placeholders [#f1]_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

Comment:
It's okay now. Thank you for writing this! I will review the section again during the release.